frontend/app/pods/admin/teams/team/edit/route.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import TeamRouteMixin from 'rafee/mixins/team-route';
+
+export default Ember.Route.extend(TeamRouteMixin, {
+
+  actions: {
+    //TODO: put this in a mixin
```

Comment:
just put it in a mixin, say like "Transition-mixin"
and just import it to every route?
no prob